src/constants/networks.js

```diff
@@ -38,6 +41,7 @@ export const TEST_NET_BINANCE_CONFIG = {
   tokenPrefix: 'b',
   env: ENVIRONMENTS.TESTNET,
   mainToken: TEST_NET_BINANCE_MAIN_TOKEN,
+  gasToken: TEST_NET_BINANCE_MAIN_TOKEN,
```

Comment:
I think gasToken is  a different token than mainToken, this looks like a copy paste error
```suggestion
  gasToken: TEST_NET_BINANCE_GAS_TOKEN,
```

Reply:
done.